mokojin/AvatarView.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  AvatarView.swift
+//  mokojin
+//
+//  Created by Assaf Gelber on 3/23/15.
+//  Copyright (c) 2015 iicninjas. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class AvatarView: UIImageView {
+    
+    var isSelected:Bool = false {
+        didSet {
+            if self.isSelected {
+                self.layer.borderWidth = 5
+                self.layer.borderColor = Constants.sharedInstance.colors.primaryColor.CGColor
```

Comment:
no need for this to be singleton.
Constants can be a struct then it can have Static constants/variables
```
struct Colors {
 static let primaryColor = ...
}
```